openmls/src/group/core_group/past_secrets.rs

```diff
@@ -109,27 +109,22 @@ impl MessageSecretsStore {
     }
 
     /// Get a mutable reference to a secret tree for a given epoch `group_epoch`.
-    /// Return a vector with the key package references and leaf indices of the
-    /// epoch.
+    /// Return a vector with the leaf indices of the epoch.
```

Comment:
Sorry if this contradicts what I said earlier, but shouldn't this really say that it returns the `MessageSecrets`, as well as a slice of the `Member`s of the epoch?

Reply:
🤦🏻 yeah, could've properly looked at it as well.